ragna/source_storages/_qdrant.py

```diff
@@ -0,0 +1,250 @@
+from __future__ import annotations
+
+import os
+import uuid
+from collections import defaultdict
+from typing import TYPE_CHECKING, Any, Optional, cast
+
+import ragna
+from ragna.core import (
+    Document,
+    MetadataFilter,
+    MetadataOperator,
+    PackageRequirement,
+    Requirement,
+    Source,
+)
+
+from ._utils import raise_no_corpuses_available, raise_non_existing_corpus
+from ._vector_database import VectorDatabaseSourceStorage
+
+if TYPE_CHECKING:
+    from qdrant_client import models
+
+
+class QdrantDB(VectorDatabaseSourceStorage):
```

Comment:
Do you want the `DB` postfix? We only did that for `LanceDB`, because that is how they have it themselves. From the Python package and docker image, I guess we should drop the postfix?

```suggestion
class Qdrant(VectorDatabaseSourceStorage):
```